source/ui/data/VerticalDelegate.js

```diff
@@ -236,12 +253,10 @@
 		* @private
 		*/
 		calculateControlsPerPage: function (list) {
-			var fn              = this[list.psizeProp],
-				multi           = list.pageSizeMultiplier || this.pageSizeMultiplier,
+			var pageSize        = this.pageSize || this.calculatePageSize(list),
```

Comment:
Will the page size always be updated as necessary here? It seems like the cached `this.pageSize` value will continue to be used even when we are in situations where we want it to be recalculated (i.e. viewport is resized).